.gitignore

```diff
@@ -1,7 +1,28 @@
+# OS Specific
 .DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+ehthumbs.db
+Thumbs.db
+
+# IDE Specific
+nbproject
+.~lock.*
+.buildpath
+.idea
+composer.lock
+*.sublime-workspace
+*.swp
+*.swo
+/.settings/
+/.project
```

Comment:
Thanks for feedback. Actually I agree with you but was unaware of global gitignore. Also I thought that lerna people are gurus compared to me so just followed what they did. Feel free to remove compose.lock as this was copy & paste from lerna's .gitignore or discard my .gitignore changes and just leave .gitignore in some "canonical state".